data/src/main/java/org/apache/iceberg/data/DeleteFilter.java

```diff
@@ -134,9 +134,13 @@ protected long pos(T record) {
 
       Iterable<CloseableIterable<Record>> deleteRecords = Iterables.transform(deletes,
           delete -> openDeletes(delete, deleteSchema));
-      StructLikeSet deleteSet = Deletes.toEqualitySet(
-          // copy the delete records because they will be held in a set
-          CloseableIterable.transform(CloseableIterable.concat(deleteRecords), Record::copy),
+
+      // copy the delete records because they will be held in a set
+      CloseableIterable<Record> records = CloseableIterable.transform(CloseableIterable.concat(deleteRecords),
+          Record::copy);
```

Comment:
Nit: it would be more readable to add a newline so that the `concat` argument and `Record::copy` are indented the same.